LEAF_Nexus/api/controllers/Group/POST.php

```diff
@@ -24,7 +24,7 @@
 });
 $this->index['POST']->register('group/[digit]/tag', function($args) use ($group) {
     try {
-        return $group->addTag($args[0], $_POST['tag']);
+        return $group->addTag($args[0], $group->sanitizeInput($_POST['tag']));
```

Comment:
There should be a `testAddTag_invalidInput()` test method to ensure the input is being sanitized properly.